maquetado/scss/book-list.scss

```diff
@@ -0,0 +1,37 @@
+.book-list {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  max-height: 768px;
+  max-width: 1366px;
```

Comment:
If you add max-width so add width: 100% too

Reply:
Done